tests/formattedcode/test_output_spdx.py

```diff
@@ -149,6 +149,8 @@ def load_and_clean_tv(location):
         line = line.strip()
         if not line:
             continue
+        if line.startswith('LicenseListVersion'):
```

Comment:
The license list version changes between `0.7.0rc0` and `0.7.0`, sorry for that. We can drop that line after the actual release again.

Reply:
thanks